src/main.rs

```diff
@@ -237,6 +239,11 @@ fn settings() -> Result<Settings, Failure> {
                 .long(DOCKER_CLI_OPTION)
                 .help("Sets the Docker CLI binary"),
         )
+        .arg(
+            Arg::with_name(FORCE_IMAGE_PULL)
+                .long(FORCE_IMAGE_PULL)
+                .help("Force pull of image from repository"),
```

Comment:
If you agree with my wording suggestion in the other comment, could you update this one too?